examples/k8s.yaml

```diff
@@ -32,9 +40,9 @@ spec:
     spec:
       containers:
       - command:
-          - "/otelsvc_agent_linux"
+          - "/usr/bin/otelcol"
           - "--config=/conf/otel-agent-config.yaml"
-        image: omnition/opencensus-agent:0.1.6
+        image: omnition/opentelemetry-collectort:v0.2.0
```

Comment:
the image name has a typo with an extra `t` at the end and should be:
```
image: omnition/opentelemetry-collector:v0.2.0
```

Reply:
Good catch - i'll update :)